engines/toon/toon.cpp

```diff
@@ -3286,7 +3289,18 @@ void ToonEngine::drawConversationLine() {
 	if (_currentTextLine && _showConversationText) {
 		_fontRenderer->setFontColorByCharacter(_currentTextLineCharacterId);
 		_fontRenderer->setFont(_currentFont);
-		_fontRenderer->renderMultiLineText(_currentTextLineX, _currentTextLineY, _currentTextLine, 0);
+		_fontRenderer->renderMultiLineText(_currentTextLineX, _currentTextLineY, _currentTextLine, 0, *_mainSurface);
+	}
+}
+
+void ToonEngine::drawCustomText(int16 x, int16 y, char *line, Graphics::Surface *frame, char color) {
```

Comment:
`byte color` is better, as you may get color index >0x7f, and `char` is a signed type on number of platforms.

`line` could also perhaps be better as `byte`, because you potentially could get characters with diacritics.

Reply:
`line` matches with the type of `_currentTextLine` (in `toon.h` 391) for consistency